Fenced_Frames_Ads_Reporting.md

```diff
@@ -324,6 +326,12 @@ window.fence.setReportEventDataForAutomaticBeacons({
 });
 ```
 
+Note that a cross-origin document calling `setReportEventDataForAutomaticBeacons` with `crossOriginExposed` will count as the document itself opting in to automatic beacons, negating the need for the document to be served with the `Allow-Fenced-Frame-Automatic-Beacons: true` header.
+
+This allows for the following use cases:
+- A root frame is served with `Allow-Cross-Origin-Event-Reporting: true`, and a cross-origin subframe sets automatic beacon data with `crossOriginExposed=true`. Automatic beacons triggered from the subframe will send with the data that was set in the subframe.
```

Comment:
Same comment about "data" here.  Perhaps replace with "`eventData`" if that's what's being referred to.